### PR TITLE
updated ai services

### PR DESCRIPTION
**Pull request by al3xne**

### CODE REVIEW

**Comment by vmchiran** [comment]
@al3xne please rename folder to follow naming convention: Everything lower case, no white spaces, instead '-'.

**Review by vmchiran — COMMENTED** [review]
@al3xne please rename folders to follow naming convention for folders and files: Everything lower case, no white spaces, instead '-'.

File: application-development/ai-cloud-services/document-understanding/README.md
Comment (vmchiran) [inline application-development/ai-cloud-services/document-understanding/README.md:1]: This file is not following the Product, Service, Workload Page Template. Please revert changes.
There is a difference between Team Publications section, which is the content we Specialists create, and the Useful Links section which can be any other Oracle public reference e.g. documentation, articles from PM etc.

Comment (AlexanderHodicke) [reply application-development/ai-cloud-services/document-understanding/README.md:1]: i changed it accordingly

**Review by AlexanderHodicke — APPROVED** [review]

**Pull request merged**